### ISSUE TITLE
CentOS: Wget is missed in hyperion-buildall.sh

### ISSUE THREAD

**Issue by N3Cr0N** [wrljet/hercules-helper#4]
Wget is missed and must be added to the hyperion-prepare.sh 

```
Hit return to continue (Step: Build Regina Rexx [used for test scripts])
/home/tester/hercules-helper/hyperion-buildall.sh: line 241: wget: command not found
```

**Comment by wrljet** [comment]
Thanks,  I will fix tonight.

**Comment by N3Cr0N** [comment]
i open a pull request 
https://github.com/wrljet/hercules-helper/pull/5/commits/d20a65fa18f00f87a059ba3632adb64c225d9a19

i live near Hamburg ;-)

**Comment by wrljet** [comment]
My mom's parents names were Gußnecker and Kilberger.

**Comment by wrljet** [comment]
Fixed with commit  b0e7ae7
(I don't know how to put that in here as a link)
